datacenter/ucp/3.0/guides/admin/install/install-on-azure.md

```diff
@@ -0,0 +1,286 @@
+    "cloudProviderBackoffRetries": 0,
+    "cloudProviderBackoffExponent": 0,
+    "cloudProviderBackoffDuration": 0,
+    "cloudProviderBackoffJitter": 0,
+    "cloudProviderRatelimit": false,
+    "cloudProviderRateLimitQPS": 0,
+    "cloudProviderRateLimitBucket": 0,
+    "useManagedIdentityExtension": false,
+    "useInstanceMetadata": true
+}
+```
+
+There are some optional values for Azure deployments:
+
+- `"primaryAvailabilitySetName": "****",` - The Worker Nodes availability set.
+- `"vnetResourceGroup": "****",` - If your Azure Network objects live in a 
+seperate resource group.
+- `"routeTableName": "****",` - If you have defined multiple Route tables within
+an Azure subnet.
+
+More details on this configuration file can be found 
+[here](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/azure/azure.go).
+
+
+
+## Considerations for IPAM Configuration
+
+The subnet and the virtual network associated with the primary interface of
+the Azure VMs need to be configured with a large enough address prefix/range. 
+The number of required IP addresses depends on the number of pods running
+on each node and the number of nodes in the cluster.
+
+For example, in a cluster of 256 nodes, to run a maximum of 128 pods
+concurrently on a node, make sure that the address space of the subnet and the
+virtual network can allocate at least 128 * 256 IP addresses, _in addition to_
+initial IP allocations to VM NICs during Azure resource creation.
+
+Accounting for IP addresses that are allocated to NICs during VM bring-up, set
+the address space of the subnet and virtual network to 10.0.0.0/16. This
+ensures that the network can dynamically allocate at least 32768 addresses,
+plus a buffer for initial allocations for primary IP addresses.
+
+> Azure IPAM, UCP, and Kubernetes
+> 
+> The Azure IPAM module queries an Azure virtual machine's metadata to obtain
+> a list of IP addresses that are assigned to the virtual machine's NICs. The
+> IPAM module allocates these IP addresses to Kubernetes pods. You configure the
+> IP addresses as `ipConfigurations` in the NICs associated with a virtual machine
+> or scale set member, so that Azure IPAM can provide them to Kubernetes when
+> requested.
+{: .important}
+
+#### Additional Notes
+
+- The `IP_COUNT` variable defines the subnet size for each node's pod IPs. This subnet size is the same for all hosts.
+-  The Kubernetes `pod-cidr` must match the Azure Vnet of the hosts. 
+
+## Configure IP pools for Azure stand-alone VMs
+
+Follow these steps once the underlying infrastructure has been provisioned.
+
+### Configure multiple IP addresses per VM NIC
+
+Follow the steps below to configure multiple IP addresses per VM NIC.
+
+1.  Initialize a swarm cluster comprising the virtual machines you created
+    earlier. On one of the nodes of the cluster, run:
+
+    ```bash
+    docker swarm init
+    ```
+
+2.  Note the tokens for managers and workers.
```

Comment:
Adding "You may retrieve the join tokens at any time by running `docker swarm join-token manager` or `docker swarm join-token worker` on the manager node" would save folks a bit of searching if their session ends or they forget to write down the token.

Reply:
Sure, done